src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/AutoRerouteV2Spec.groovy

```diff
@@ -151,6 +152,29 @@ class AutoRerouteV2Spec extends HealthCheckSpecification {
             assert flowInfo.statusInfo == "Switch $sw.dpId is inactive"
         }
 
+        when: "Other isl fails"
+        def otherIslFailed = false
+        def otherIsl = topology.isls.find() {isl-> isl.srcSwitch != sw && isl.dstSwitch != sw}
+        antiflap.portDown(otherIsl.srcSwitch.dpId, otherIsl.srcPort)
+        otherIslFailed = true
+        Wrappers.wait(WAIT_OFFSET) {
+            assert northbound.getLink(otherIsl).actualState == IslChangeType.FAILED
```

Comment:
Why do you check the `actualState` field?
probably would be better to check the `state` field